references/obj_detection/train_pytorch.py

```diff
@@ -168,7 +195,7 @@ def main(args):
     max_score = 0.
 
     for epoch in mb:
-        fit_one_epoch(model, train_loader, optimizer, scheduler, mb)
+        fit_one_epoch(model, train_loader, optimizer, scheduler, mb, is_transform=True)
```

Comment:
that doesn't sound like an explicit argument name :sweat_smile:

Reply:
Thanks, I have changed it, and included it in argument parser with a default bool value set to True.
 `fit_one_epoch(model, train_loader, optimizer, scheduler, mb, use_aug=args.use_augmentations)`